packages/server/lib/browsers/cri-client.ts

```diff
@@ -109,21 +119,52 @@ export const create = async (debuggerUrl: websocketUrl): Promise<CRIWrapper> =>
 
   maybeDebugCdpMessages(cri)
 
+  let cachedProtocolVersionP
+
+  const ensureMinimumProtocolVersion = (protocolVersion: string): Promise<void> => {
+    return getProtocolVersion()
+    .then((actual) => {
+      const minimum = getMajorMinorVersion(protocolVersion)
+
+      const hasVersion = actual.major > minimum.major
+         || (actual.major === minimum.major && actual.minor >= minimum.minor)
```

Comment:
dense logic, extract to a named function for clarity

Reply:
b2323e059